popin-form.php

```diff
@@ -0,0 +1,32 @@
+<?php get_header(); ?>
+
+<form>
+  <div class="poppin-namesurname">
+	  <p>First name:<br /></p>
+	  <input type="text" name="firstname">
+	  <br />
+	  <p>Last name:<br /></p>
+	  <input type="text" name="lastname">
+	  <br>
```

Comment:
Also here don't use <br/> for spacing/layout. 

You can rewrite this as follows:

```
<ul class='form-fields'>
  <li>
      <label for="firstname">First name:</label>
      <input type="text" name="firstname">
  </li>
  <li>
      <label for="lastname">Last name:</label>
      <input type="text" name="lastname">
    </li>
</ul>
```

To be semantically correct you need labels for your input fields.